services/ip_cam_trendnet_tv_ip100/ip_cam_trendnet_tv_ip100_service.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""Honeycomb IP Cam TRENDnet TV-IP100 Service."""
+from __future__ import unicode_literals
+
+import os
+
+from six.moves.urllib import parse as urlparse
+from six.moves.BaseHTTPServer import HTTPServer
+from six.moves.SimpleHTTPServer import SimpleHTTPRequestHandler
+from six.moves.socketserver import ThreadingMixIn
+
+import requests
+
+from honeycomb.servicemanager.base_service import ServerCustomService
+
+DEFAULT_PORT = 80
```

Comment:
if port parameter is required there's no need for DEFAULT_PORT

Reply:
It is needed for testing

Reply:
the port in test args is hardcoded so the port in `test()` can also be hardcoded
or change the const name to TEST_PORT